WORKING_GROUPS.md

```diff
@@ -205,6 +208,47 @@ Their responsibilities are:
 The current members can be found in their
 [README](https://github.com/iojs/nan#collaborators).
 
+### [Benchmarking](https://github.com/nodejs/benchmarking)
+
+The Benchmark working group's purpose is to gain consensus for an agreed set of benchmarks that can be used to:
+
+1. track and evangelize performance gains made between Node releases
+2. avoid performance regressions between releases
+
+Its responsibilities are:
+
+1. Identify 1 or more benchmarks that reflect customer usage.  Likely need more than one to cover typical Node use cases including low-latency and high concurrency
+2. Work to get community consensus on the list chosen
+3. Add regular execution of chosen benchmarks to Node builds
+4. Track/publicize performance between builds/releases
+
+
+AcmeAir is proposed as one of the benchmark to consider: https://github.com/acmeair/acmeair-nodejs
```

Comment:
does this line need to be included here? surely this is detail that should be left to that WG rather than cementing it here where it's harder to modify